en_US/data-integration/data-bridge-mqtt.md

```diff
@@ -47,9 +47,9 @@ Before creating an MQTT Broker data integration, you need to obtain the connecti
 
 The data integration provides good compatibility and support for EMQX or other standard MQTT servers. If you need to connect to other types of MQTT services, you can refer to their relevant documentation to obtain the connection information. Generally, most IoT platforms provide standard MQTT access methods, and you can convert device information into the aforementioned MQTT connection information based on their guidance.
 
-:::tip Note 
+:::tip Note
 
-When EMQX is running in cluster mode or when a connection pool is enabled, using the same client ID to connect multiple nodes to the same MQTT service usually leads to device conflicts. Therefore, the MQTT message bridge currently does not support setting a fixed client ID. 
+When EMQX is running in cluster mode or when a connection pool is enabled, using the same client ID to connect multiple nodes to the same MQTT service usually leads to device conflicts. Therefore, the MQTT message bridge currently does not support setting a fixed client ID.
```

Comment:
This pr is talking about how to set the static (fixed?) client ID, so is this "Note" still valid? @thalesmg @HJianBo